canonical/ubuntu.com
# Update the Ubuntu Pro description to the latest version

## Done

- Update the description to the latest version 9th Nov.

## QA

- Open the demo
- Go to /legal/ubuntu-pro
- Click "View the Ubuntu Pro service description ›"
- See the newly designed asset
